http_prompt/commandio.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import six
+
+from .printer import Printer
+from .utils import strip_color_codes
+
+
+class CommandIO():
```

Comment:
Actually, I just realized `CommandIO` is to provide an abstraction where file output needs `strip_color_codes` and stdout doesn't. HTTPie has taken care of the same problem and I think we should take advantage of that. So when users enter `post > file.txt`, what we should do is tell httpie to write its output to the file. However, we still have to take care of output redirection ourselves when the command is `env` as in `env > file.txt`.